pkg/resources/management.cattle.io/v3/globalrolebinding/validator.go

```diff
@@ -220,3 +223,21 @@ func (a *admitter) validateGlobalRole(globalRole *v3.GlobalRole, fieldPath *fiel
 	}
 	return nil
 }
+
+func (a *admitter) validateNamespacedRules(request *admission.Request, rules map[string][]v1.PolicyRule) []error {
+	var returnErrs []error
+	// check for escalation in each namespace of the GlobalRole's NamespacedRules
+	for namespace, rules := range rules {
+		// check that namespace exists
+		_, err := a.namespaces.Get(namespace)
+		if err != nil {
+			returnErrs = append(returnErrs, fmt.Errorf("error getting namespace %v: %w", namespace, err))
+			continue
+		}
```

Comment:
I think the validation for this part should probably be on the GlobalRole rather than the binding (if it is anywhere).